### DISCUSSION TITLE
Sending the reading as photo

### DISCUSSION THREAD

**Discussion by chobott** [jomjol/AI-on-the-edge-device#1123] [Ideas]
Hi there!

we are looking for best way how i can send every month RAW Value with Photo to waterworks services. If ESP can send directly RAW value with attached Picture or use Home Assistant?

Tom

**Comment by jomjol** [comment]
Hi Tom,
that sound like a commercial application. Can you please contact me ( mueller.josef(@)gmail.com ). 
Thanks, jomjol

**Reply by chobott** [reply]
It is primary for my home :-) email send... Tom

**Comment by caco3** [comment]
You always can access the images through `http://<IP>/img_tmp/alg_roi.jpg` resp `http://<IP>/img_tmp/raw.jpg`.
The sending then can happen with Homeassistant or another tool.
To access the parsed values, see https://github.com/jomjol/AI-on-the-edge-device/wiki/Integration-Home-Assistant

See also: https://github.com/jomjol/AI-on-the-edge-device/issues/1102

**Reply by chobott** [reply]
Yes....probably i can use this way, and read RAW to number and send email....but other people can use standalone ESPcam for commercial like write jomjol......

**Reply by Terr4** [reply]
> You always can access the images through `http://<IP>/img_tmp/alg_roi.jpg` resp `http://<IP>/img_tmp/raw.jpg`.

Hello! I also try to access the picture as some people in my household do not trust the image recognition (haha)
I tried to access those URLs but I get no response.

Also when I look at the "img_tmp" folder in the Fileserver UI I cannot find those pictures?
Fileserver UI only lists:

leer.txt
main_dig1.bmp
main_dig2.bmp
main_dig3.bmp
main_ana1.bmp
main_ana2.bmp
main_ana3.bmp
main_ana4.bmp
ref0.jpg
ref0_org.jpg

**Reply by jomjol** [reply]
The images are send on the fly from the memory.

**Reply by jomjol** [reply]
The refx.jpg are stored in the config folder.

**Reply by Terr4** [reply]
> The images are send on the fly from the memory.

Thank you! It works now with the link.
I copy the file now every 5min to another PC with:
`wget -O "/etc/openhab/html/wasserzaehler.jpg" "http://<ipofwasserzaehler>/img_tmp/alg_roi.jpg"`
